Net/src/TCPServerDispatcher.cpp

```diff
@@ -147,16 +147,8 @@ void TCPServerDispatcher::enqueue(const StreamSocket& socket)
 		_queue.enqueueNotification(new TCPConnectionNotification(socket));
 		if (!_queue.hasIdleThreads() && _currentThreads < _pParams->getMaxThreads())
 		{
-			try
-			{
-				_threadPool.startWithPriority(_pParams->getThreadPriority(), *this, threadName);
-				++_currentThreads;
-			}
-			catch (Poco::Exception&)
-			{
-				// no problem here, connection is already queued
-				// and a new thread might be available later.
-			}
+            _threadPool.startWithPriority(_pParams->getThreadPriority(), *this, threadName);
+            ++_currentThreads;
 		}
```

Comment:
Does the threadPool throw if the pool is full (and should be caught)?  Or does it simply do nothing?